### DISCUSSION TITLE
Rozdzielenie street w checkout-form na streetname i buildingnumber

### DISCUSSION THREAD

**Discussion by q3ok** [allegro/allegro-api#9218] [Suggestions - rejected]
https://github.com/allegro/allegro-api/issues/9214

Aktualnie checkout-form podaje nazwę ulicy i numer budynku w jednym parametrze.
Zasób do generowania wysyłki w WzA wymaga rozdzielonych pól.
Rozdzielenie tego w niektórych wypadkach może być bardzo trudne, a wręcz niemożliwe bez ręcznej weryfikacji użytkownika.

Proponuję, by już na etapie checkout-form te pola zostały rozdzielone. Z racji zgodności wstecznej najlepiej gdyby były dostępne trzy pola w checkout-form, tj. 
1: nazwa ulicy z numerem budynku (jak dotychczas) -> jako deprecated?
2: sama nazwa ulicy
3: nazwa ulicy z numerem budynku.

**Comment by PrzemyslawLukanowski** [comment]
Zaktualizowaliśmy status sugestii jako "odrzucona", ponieważ aktualnie w zasobach Wysyłam z Allegro nie wymagamy rozdzielenia tych pól. Udostępniliśmy tak że nowy zasób [GET /shipment-management/delivery-proposals/{orderId}](https://developer.allegro.pl/documentation/#tag/Shipment-management/operation/getDeliveryProposals), dzięki któremu możecie pobrać dokładne dane potrzebne do utworzenia przesyłki.
